src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -200,16 +200,6 @@ public static String formatDate(Date date) {
         sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
         return sdf.format(date);
     }
-
-    /**
-     * Formats a date in the format dd MMM yyyy, HH:mm. Example: 05 May 2012,
-     * 22:04<br />
-     */
-    public static String formatTime(Date date) {
-        if (date == null)
-            return "";
-        return new SimpleDateFormat("EEE, dd MMM yyyy, HH:mm").format(date);
```

Comment:
Not relevant to the diff, but I'm just wondering, if the time zone is not set, what time zone will `SimpleDateFormat` operate on?